README.md

```diff
@@ -84,7 +84,7 @@ Turtle aims to support full syntax of the Ninja build files. It also supports ba
 - [ ] C/C++ header dependencies
   - [ ] `depfile` option
   - [ ] `deps` option
-- [ ] Windows support
+- [x] Windows support
```

Comment:
Just so you know, we rather need to run commands in rules directly rather than wrapping them in `sh` for the full Windows support. So for now, it might work in an unexpected way. https://ninja-build.org/manual.html#ref_rule

Reply:
Yeah. I didn't know that turtle does that. We should probably open an issue,